.github/workflows/integ_tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Integration tests
+on: [push]
+
+jobs:
+  bat:
+    name: Run Integ Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12"
+      - name: Perform npm tasks
+        run: npm run ci
+
+      - name: Perform 'setup-matlab'
+        uses: matlab-actions/setup-matlab@v0
+        
+      - name: Generate Model
+        uses: matlab-actions/run-command@v0
+        with: 
+            command: cd('sample'),createSampleModel('temp_model'),sltest.testmanager.createTestsFromModel('sltestfile', 'temp_model');
+
+      - name: Run tests
+        uses: ./
+        with:
+            test-results-simulink-test: test-results/simulinktest.mldatx
+            model-coverage-cobertura: test-results/modelcoverage.xml
+            test-results-pdf: test-results/pdf-results.pdf
+            select-by-folder: sample
+
+      - name: Verify artifacts generation
+        run: |
+            (test -f test-results/simulinktest.mldatx) || (echo "File simulinktest.mldatx does not exist" && exit 1)
+            (test -f test-results/modelcoverage.xml) || (echo "File modelcoverage.xml does not exist" && exit 1)
+            (test -f test-results/pdf-results.pdf) || (echo "File pdf-results.pdf does not exist" && exit 1)
+
```

Comment:
I think this is a great integration test 😄 
But I do believe it is important that we also retain the original test too since it omitted most of the non-required options. This way we know that this action will work even without the non-required options specified 🤔